frontend/src/Explorer/I18n/EN.purs

```diff
@@ -37,6 +37,7 @@ translation =
         , cHours: "Hours"
         , cMinutes: "Minutes"
         , cSeconds: "Seconds"
+        , cDateFormat: "MM/DD/YYYY HH:mm:ss"
```

Comment:
Can the date format really be different for different languages? 
It's a small detail, but maybe put it in `Explorer.Types.State` if you want, which can then be used to change formats while the application is running (could be a requirement) or test it with a different format.